CLI: Fix flaky test ContentTypeCommandIT.Test_Command_Content_Type_Folder_Push

### Problem Statement

The test fails sporadically with this error:

```
org.opentest4j.AssertionFailedError: expected: <0> but was: <1>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:527)
	at com.dotcms.cli.command.contenttype.ContentTypeCommandIT.Test_Command_Content_Type_Folder_Push(ContentTypeCommandIT.java:1145)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.quarkus.test.junit.QuarkusTestExtension.runExtensionMethod(QuarkusTestExtension.java:1013)
	at io.quarkus.test.junit.QuarkusTestExtension.interceptTestMethod(QuarkusTestExtension.java:827)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	Suppressed: com.dotcms.api.client.pull.exception.PullException: 404
		at com.dotcms.api.client.util.ErrorHandlingUtilImpl.mapException(ErrorHandlingUtilImpl.java:198)
		at com.dotcms.api.client.util.ErrorHandlingUtilImpl.mapPullException(ErrorHandlingUtilImpl.java:175)
		at com.dotcms.api.client.util.ErrorHandlingUtilImpl_ClientProxy.mapPullException(Unknown Source)
		at com.dotcms.api.client.pull.PullServiceImpl.fetch(PullServiceImpl.java:151)
		at com.dotcms.api.client.pull.PullServiceImpl.pull(PullServiceImpl.java:48)
		at com.dotcms.api.client.pull.PullServiceImpl_Subclass.pull$$superforward(Unknown Source)
		at com.dotcms.api.client.pull.PullServiceImpl_Subclass$$function$$1.apply(Unknown Source)
		at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:73)
		at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:62)
		at io.quarkus.arc.impl.ActivateRequestContextInterceptor.invoke(ActivateRequestContextInterceptor.java:124)
		at io.quarkus.arc.impl.ActivateRequestContextInterceptor.aroundInvoke(ActivateRequestContextInterceptor.java:33)
		at io.quarkus.arc.impl.ActivateRequestContextInterceptor_Bean.intercept(Unknown Source)
		at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:42)
		at io.quarkus.arc.impl.AroundInvokeInvocationContext.perform(AroundInvokeInvocationContext.java:30)
		at io.quarkus.arc.impl.InvocationContexts.performAroundInvoke(InvocationContexts.java:27)
		at com.dotcms.api.client.pull.PullServiceImpl_Subclass.pull(Unknown Source)
		at com.dotcms.cli.command.contenttype.ContentTypePull.call(ContentTypePull.java:97)
		at com.dotcms.cli.command.contenttype.ContentTypePull_Subclass.call$$superforward(Unknown Source)
		at com.dotcms.cli.command.contenttype.ContentTypePull_Subclass$$function$$4.apply(Unknown Source)
		at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:73)
		at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:62)
		at io.quarkus.arc.impl.ActivateRequestContextInterceptor.invoke(ActivateRequestContextInterceptor.java:124)
		at io.quarkus.arc.impl.ActivateRequestContextInterceptor.aroundInvoke(ActivateRequestContextInterceptor.java:33)
		at io.quarkus.arc.impl.ActivateRequestContextInterceptor_Bean.intercept(Unknown Source)
		at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:42)
		at io.quarkus.arc.impl.AroundInvokeInvocationContext.perform(AroundInvokeInvocationContext.java:30)
		at io.quarkus.arc.impl.InvocationContexts.performAroundInvoke(InvocationContexts.java:27)
		at com.dotcms.cli.command.contenttype.ContentTypePull_Subclass.call(Unknown Source)
		at com.dotcms.cli.command.contenttype.ContentTypePull.call(ContentTypePull.java:24)
		at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
		at picocli.CommandLine.access$1500(CommandLine.java:148)
		at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
		at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
		at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
		at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
		at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
		at com.dotcms.cli.common.DotExecutionStrategy.internalExecute(DotExecutionStrategy.java:183)
		at com.dotcms.cli.common.DotExecutionStrategy.execute(DotExecutionStrategy.java:94)
		at picocli.CommandLine.execute(CommandLine.java:2170)
		at com.dotcms.cli.command.contenttype.ContentTypeCommandIT.Test_Command_Content_Type_Folder_Push(ContentTypeCommandIT.java:1143)
		... 5 more
	Caused by: jakarta.ws.rs.NotFoundException: 404
		at com.dotcms.api.provider.DefaultResponseExceptionMapper.toThrowable(DefaultResponseExceptionMapper.java:62)
		at com.dotcms.api.provider.DefaultResponseExceptionMapper.toThrowable(DefaultResponseExceptionMapper.java:21)
		at org.jboss.resteasy.microprofile.client.ExceptionMapping$HandlerException.mapException(ExceptionMapping.java:60)
		at io.quarkus.restclient.runtime.QuarkusProxyInvocationHandler.invoke(QuarkusProxyInvocationHandler.java:172)
		at jdk.proxy5/jdk.proxy5.$Proxy210.getContentType(Unknown Source)
		at com.dotcms.api.client.pull.contenttype.ContentTypeFetcher.fetchByKey(ContentTypeFetcher.java:61)
		at com.dotcms.api.client.pull.contenttype.ContentTypeFetcher_Subclass.fetchByKey$$superforward(Unknown Source)
		at com.dotcms.api.client.pull.contenttype.ContentTypeFetcher_Subclass$$function$$1.apply(Unknown Source)
		at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:73)
		at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:62)
		at io.quarkus.arc.impl.ActivateRequestContextInterceptor.invoke(ActivateRequestContextInterceptor.java:124)
		at io.quarkus.arc.impl.ActivateRequestContextInterceptor.aroundInvoke(ActivateRequestContextInterceptor.java:33)
		at io.quarkus.arc.impl.ActivateRequestContextInterceptor_Bean.intercept(Unknown Source)
		at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:42)
		at io.quarkus.arc.impl.AroundInvokeInvocationContext.perform(AroundInvokeInvocationContext.java:30)
		at io.quarkus.arc.impl.InvocationContexts.performAroundInvoke(InvocationContexts.java:27)
		at com.dotcms.api.client.pull.contenttype.ContentTypeFetcher_Subclass.fetchByKey(Unknown Source)
		at com.dotcms.api.client.pull.contenttype.ContentTypeFetcher.fetchByKey(ContentTypeFetcher.java:17)
		at com.dotcms.api.client.pull.PullServiceImpl.lambda$fetch$0(PullServiceImpl.java:109)
		at io.smallrye.context.impl.wrappers.SlowContextualSupplier.get(SlowContextualSupplier.java:21)
		at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
		at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:587)
		at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
		at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538)
		at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
		at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
		at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
		at java.base/java.lang.Thread.run(Thread.java:1583)
```

https://github.com/dotCMS/core/actions/runs/11220759588/job/31193073549

--- a/tools/dotcms-cli/cli/src/test/java/com/dotcms/cli/common/ContentTypesTestHelperService.java
+++ b/tools/dotcms-cli/cli/src/test/java/com/dotcms/cli/common/ContentTypesTestHelperService.java
@@ -63,9 +63,6 @@ public class ContentTypesTestHelperService {
 
         final ContentTypeAPI contentTypeAPI = clientFactory.getClient(ContentTypeAPI.class);
 
-        final long identifier = System.currentTimeMillis();
-        final String varName = "var_" + identifier;
-
         final ImmutableSimpleContentType contentType = buildContentType(
                 null, null, detailPage, urlMapPattern
         );
@@ -74,7 +71,12 @@ public class ContentTypesTestHelperService {
                 from(contentType).build();
         contentTypeAPI.createContentTypes(List.of(saveRequest));
 
-        return varName;
+        // Make sure the content type is created, and we are giving the server some time to process
+        findContentType(contentType.variable()).orElseThrow(() -> new RuntimeException(
+                "Content type not found after creation: " + contentType.variable()
+        ));
+
+        return contentType.variable();
     }
 
     /**

